### PR TITLE
Add `sensitive` parameter to `copyable`

### PR DESCRIPTION
**Pull request by GuillaumeRx**
This adds a new optional parameter to the `copyable` component.

### CODE REVIEW

File: packages/snaps-ui/src/builder.test.ts [hunk 1]
```diff
@@ -16,9 +16,10 @@ describe('copyable', () => {
       value: 'Hello, world!',
     });
 
-    expect(copyable({ value: 'foo bar' })).toStrictEqual({
+    expect(copyable({ value: 'foo bar', sensitive: true })).toStrictEqual({
```
Comment (Mrtenz) [inline packages/snaps-ui/src/builder.test.ts:19]: Can you add a case without, and this set to false too?

Comment (GuillaumeRx) [reply packages/snaps-ui/src/builder.test.ts:19]: There's a case without it right above ;) I'm adding a case with it set to `false`

Comment (GuillaumeRx) [reply packages/snaps-ui/src/builder.test.ts:19]: f8a8b13

File: packages/snaps-ui/src/builder.ts [hunk 2]
```diff
@@ -102,10 +102,12 @@ function createBuilder<
  * @param args - The node arguments. This can either be a string, or an object
  * with the `text` property.
  * @param args.text - The text to copy.
+ * @param args.sensitive - Wether if the value is sensitive or not. Defaults to false.
```
Comment (Mrtenz) [inline packages/snaps-ui/src/builder.ts:105]:
```suggestion
 * @param args.sensitive - Whether the value is sensitive or not. Defaults to false.
```

File: packages/snaps-ui/src/nodes.ts [hunk 3]
```diff
@@ -80,6 +81,7 @@ export const CopyableStruct = assign(
  *
  * @property type - The type of the node, must be the string 'copyable'.
  * @property value - The text to be copied.
+ * @property sensitive - Wether if the value is sensitive or not. Defaults to false.
```
Comment (Mrtenz) [inline packages/snaps-ui/src/nodes.ts:84]:
```suggestion
 * @property sensitive - Whether the value is sensitive or not. Defaults to false.
```

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**